src/main/kotlin/com/github/hd/tornadofxsuite/view/Dialog.kt

```diff
@@ -28,20 +28,18 @@ class Dialog : Fragment() {
                     vboxConstraints { marginRight = 5.0 }
 
                     listview<String> {
-                        if (fxTestGenerator.scanner.mapClassViewNodes.size > 0) {
-                            fxTestGenerator.scanner.mapClassViewNodes.forEach { (className, digraph) ->
-                                items.add(className)
-                                digraph.viewNodes.forEach { (bucket, children) ->
-                                    val nodeLevel = bucket.level
-                                    var viewNode = "$nodeLevel \t${bucket.nodeType}"
+                        fxTestGenerator.scanner.mapClassViewNodes.forEach { (className, digraph) ->
+                            items.add(className)
+                            digraph.viewNodes.forEach { (bucket, children) ->
+                                val nodeLevel = bucket.level
+                                var viewNode = "$nodeLevel \t${bucket.nodeType}"
```

Comment:
This one I have to pull down and double check - I feel like I put that change in there at some point when I think the listview was freaking out about an empty input!